samples/net/mqtt_publisher/src/main.c

```diff
@@ -371,10 +363,7 @@ static void publisher(void)
 	SUCCESS_OR_EXIT(rc);
```

Comment:
Too bad I cannot leave the comment in any place in this file...

Anyway, we need to move `tls_init` function call out of the `publisher` function and the while loop, It should be called only once for reboot, otherwise it throws errors on re-initialization.